Document hf.co webhook publicly (#156)
huggingface/huggingface_hub: docs/hub/endpoints.md

/api/logout  <br/>                 POST                                          | Log out user.                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | `logout(token)`                        | ```headers = { authorization :  "Bearer $token" }```                                                                                     |

## Webhook

> If you ever need to programmatically get notified about all changes/additions to models on the Hugging Face Hub, you can subscribe to the Hub webhook.

Reviewer: Nits

```suggestion
If you ever need to programmatically get notified about all the changes/additions to model repositories on the Hugging Face Hub, you can subscribe to the Hugging Face Hub webhook.
```

> When you're subscribed – meaning you sent us a URL on your side, that you want us to ping – we will call it over HTTP with the following payload:

Reviewer: More nits

```suggestion
When you're subscribed – meaning you sent us a URL on your side that you want us to ping – we will call it over HTTP with the following payload:
```

```python

> { "add": "user/model_id" }

Reviewer: Should we add one example with organization/model_id?

# or

> { "update": "user/model_id" }

Author: ```suggestion
{ "update": "organization/model_id" }
```
